src/sagas/index.js

```diff
@@ -1,8 +1,24 @@
 'use strict'
 
-const { call, put, takeLatest, all } = require('redux-saga/effects')
+const url = require('url')
+const promisify = require('es6-promisify')
+const request = promisify(require('request'))
 
-const { listObservations, importSurvey } = require('../drivers/local')
+const {
+  call,
+  put,
+  takeLatest,
+  takeEvery,
+  all
+} = require('redux-saga/effects')
+
+const {
+  listObservations,
+  importSurvey,
+  importOsm
+} = require('../drivers/local')
+
+const OSMAPI = 'http://api.openstreetmap.org/api/0.6/'
```

Comment:
Let's make this configurable earlier-on (potentially via environment var for now?).